carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/report/TestResultItem.java

```diff
@@ -89,10 +69,7 @@ public String getFailReason() {
 
     }
 
-    public void setFailReason(String failReason) {
-        this.failReason = failReason;
-    }
-
+    //TODO: move description init into the constructor
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1135)